spec/models/user_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+
+  it "create users" do
+    user1 = User.create({name:'Pasha', email:'test@mail.ru', password: 'testtest'})
```

Comment:
It's better not to create local variables inside specs and instead use `let` and `subject` directives:
- https://www.betterspecs.org/#let
- https://www.betterspecs.org/#subject